README.md

```diff
@@ -263,6 +263,16 @@ govalid consistently outperforms reflection-based validators by **5x to 44x**:
 
 [📊 View Complete Benchmarks](test/benchmark/README.md)
 
+### 🔧 Development Setup
+
+For contributors, install lefthook to enable pre-commit checks:
+
+```bash
+make install-lefthook
+```
+
+This will install and configure lefthook to run automatic checks before each commit, ensuring code quality and consistency.
```

Comment:
```suggestion
Because of this, lefthook is installed, then the code-base would be checked automatically before each commit, ensuring code quality and consistency.
```